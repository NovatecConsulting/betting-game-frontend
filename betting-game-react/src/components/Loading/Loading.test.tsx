import React from "react";
import { shallow } from "enzyme";
import { Loading } from ".";

describe(Loading.name + " component", () => {
    it("should render and match snapshot.", () => {
        const wrapper = shallow(<Loading />);

        expect(wrapper).toIncludeText("Loading...");
        expect(wrapper).toMatchSnapshot();
    });

    it("should render and match snapshot with size attribute set.", () => {
        const wrapper = shallow(<Loading size={32} />);

        expect(wrapper).toIncludeText("Loading...");
        expect(wrapper).toMatchSnapshot();
    });
});
