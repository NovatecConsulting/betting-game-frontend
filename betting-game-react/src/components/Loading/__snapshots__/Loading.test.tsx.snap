// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render and match snapshot with size attribute set. 1`] = `
<div
  className="flex justify-center mt-8"
>
  <div
    className="loader ease-linear rounded-full border-4 border-t-4 border-gray-200 h-32 w-32"
  />
  <span
    className="sr-only"
  >
    Loading...
  </span>
</div>
`;

exports[`Loading component should render and match snapshot. 1`] = `
<div
  className="flex justify-center mt-8"
>
  <div
    className="loader ease-linear rounded-full border-4 border-t-4 border-gray-200 h-16 w-16"
  />
  <span
    className="sr-only"
  >
    Loading...
  </span>
</div>
`;
